### PR TITLE
New Package: ArchiMateTool 5.2.0

### PR DESCRIPTION
**Pull request by yoshimov**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135652)

### CODE REVIEW

**Comment by yoshimov** [comment]
@microsoft-github-policy-service agree

File: manifests/a/ArchiMateTool/Archi/5.2.0/ArchiMateTool.Archi.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ArchiMateTool.Archi
+PackageVersion: 5.2.0
+PackageLocale: en-US
+Publisher: Phillip Beauvoir & Jean-Baptiste Sarrodie
+PackageName: Archi
+License: MIT License
+Copyright: Phillip Beauvoir
+ShortDescription: Archi Setup
```
Comment (Trenly) [inline manifests/a/ArchiMateTool/Archi/5.2.0/ArchiMateTool.Archi.locale.en-US.yaml:11]: Okease provide a better description of what this package is, does, or is used for

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/a/ArchiMateTool/Archi/5.2.0/ArchiMateTool.Archi.locale.en-US.yaml (hunk 1 above)
Comment (yoshimov) [reply manifests/a/ArchiMateTool/Archi/5.2.0/ArchiMateTool.Archi.locale.en-US.yaml]: > Okease provide a better description of what this package is, does, or is used for

Thank you for your comment. I just updated description of Archi.

**Review by stephengillie — APPROVED** [review]

**Comment by mdanish-kh** [comment]
@yoshimov  You'll have to mark the above conversation as "resolved" for auto-merge process to kick in...

**Pull request merged**